habitat-lab/habitat/tasks/nav/object_nav_task.py

```diff
@@ -183,3 +183,9 @@ class ObjectNavigationTask(NavigationTask):
     r"""An Object Navigation Task class for a task specific methods.
     Used to explicitly state a type of the task in config.
     """
+
+@registry.register_task(name="ObjectNav-v2")
+class ObjectNavigationTaskV2(ObjectNavigationTask):
```

Comment:
Is this new class necessary ?

Reply:
The idea by @devendrachaplot was that having a separate version for the current task will allow us others to refer to this task with a simple name. Otherwise people will be calling it continuous control objectnav which is much more verbose.